evaluation/swe_bench/README.md

```diff
@@ -44,7 +44,6 @@ max_iterations = 100
 cache_dir = "/tmp/cache"
 sandbox_type = "ssh"
 ssh_hostname = "localhost"
-sandbox_timeout = 120
```

Comment:
Please let's leave these in place in the documentation of evals. 2 minutes is default today, yes, but it might or not be tomorrow. For evals, though, reproducibility of results is essential.